src/devices_models/devices/common/range_constraint.jl

```diff
@@ -53,34 +117,55 @@ function add_range_constraints!(
     time_steps = get_time_steps(container)
     device_names = [PSY.get_name(d) for d in devices]
 
-    con_ub = add_cons_container!(
+    con_lb = add_cons_container!(
         container,
         constraint,
         component_type,
         device_names,
         time_steps,
-        meta = "ub",
+        meta = "lb",
     )
-    con_lb = add_cons_container!(
+
+    for (i, device) in enumerate(devices), t in time_steps
+        ci_name = PSY.get_name(device)
+        # TODO: deal with additional expressions terms for services
+        limits = get_min_max_limits(device, T, W) # depends on constraint type and formulation type
```

Comment:
If we are splitting this function in lb and ub we should re-implement the `get_min_max_limits` so we don't need to call the methods twice